guides/common/modules/ref_capsule-storage-requirements.adoc

```diff
@@ -11,8 +11,7 @@ The runtime size was measured with Red{nbsp}Hat Enterprise Linux 6, 7, and 8 rep
 [cols="1,1,1",options="header"]
 |====
 |Directory |Installation Size |Runtime Size
-|/var/cache/pulp/ |1 MB | 20 GB (Minimum)
 |/var/lib/pulp/ |1 MB |300 GB
-|/var/spool/squid/ |0 GB |10 GB
+|/var/opt/rh/rh-postgresql12/lib/pgsql |100 MB |10 GB
```

Comment:
I noticed a mismatch here between satelltie storage and capsule storage (20 GB vs 10GB), should we make these the same?

Reply:
Server storage includes Foreman and Candlepin databases as well, thus making the sizing requirements larger than a pure proxy with only Pulp 3.